docs/user-manual.md

```diff
@@ -59,8 +59,30 @@ In addition to the properties associated with a specific particle, the tool allo
 Once computed for the first time, the magnification parameters are stored and used to convert all measurements. If you need to recompute the magnification parameters, you can do so by clicking on the `Measure magnification` button again. The tool will remember previously computed magnification parameters, and will allow you to switch between them.
 
 ### Saving the data
-To save the data, click on the `Save to file` button. This will open a file dialog, which will allow you to select the file where you want to save the data. The tool will save the data in a CSV file, which will contain the information about the particles, and their properties, as well as the magnification parameters.
+The data is stored internally as a list of `ParticleDecays` objects, which contain the information about the particles, their properties, as well as the magnification parameters.
 
+To save the data to a file for the analysis, click on the `Save to file` button. This will open a file dialog, which will allow you to select the file where you want to save the data. Two file types are suported: `CSV` and `Pickle`.
+
+The `CSV` file will save the data in a human-readable format, which can be opened in a text editor or a spreadsheet program. Make sure to import the data as a CSV file with the correct delimiter (`,`) and the right encoding (`UTF8`) so that the symbols are rendered correctly.
```

Comment:
```suggestion
`CSV` format is a readable comma-separated file format, which can be opened in a text editor, or a spreadsheet program, or using some other analysis tool like [pandas](https://pandas.pydata.org/). Make sure to import the data as a CSV file with the correct delimiter (`,`) and the correct encoding (`UTF8`) so that the symbols are rendered correctly.
```